tracim/migration/versions/69fb10c3d6f0_files_on_disk.py

```diff
@@ -0,0 +1,25 @@
+"""files on disk
+
+Revision ID: 69fb10c3d6f0
+Revises: c1cea4bbae16
+Create Date: 2017-06-07 17:25:47.306472
+
+"""
+
+from alembic import op
+from depot.fields.sqlalchemy import UploadedFileField
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '69fb10c3d6f0'
+down_revision = 'c1cea4bbae16'
+
+
+def upgrade():
+    op.add_column('content_revisions',
+                  sa.Column('depot_file',
```

Comment:
Here what it produces in SQLite3:
```
sqlite> .schema content_revisions
[...]
        depot_file VARCHAR(4000),
[...]
sqlite> select depot_file from content_revisions;
{"files": ["default/fdb74f3c-5daa-11e7-a329-d8cb8a1abe16"],
 "depot_name": "default",
 "_public_url": null,
 "filename": "unnamed",
 "path": "default/fdb74f3c-5daa-11e7-a329-d8cb8a1abe16",
 "file_id": "fdb74f3c-5daa-11e7-a329-d8cb8a1abe16",
 "content_type": "application/octet-stream",
 "uploaded_at": "2017-06-30 15:44:24"}
```
that is to say a python dictionary of file's metadata.